src/mapeo-project.js

```diff
@@ -626,9 +628,28 @@ export class MapeoProject extends TypedEmitter {
       ])
     )
 
-    // TODO: 3. Clear data from indexes
+    // 3. Clear data from indexes
     // 3.1 Reset multi-core indexer state
+    await Promise.all(
+      Object.values(this.#dataStores)
+        .filter((dataStore) => dataStore.namespace !== 'auth')
+        .map(async (dataStore) => {
+          await dataStore.close()
+          await dataStore.unlink()
+        })
+    )
+
     // 3.2 Clear indexed data
+    const isIndexedSchema = setHas(new Set(this.#indexWriter.schemas))
+    await Promise.all(
+      Object.values(this.#dataTypes)
+        .filter((dataType) =>
+          namespacesWithoutAuth.includes(dataType.namespace)
```

Comment:
nitpick, but this being different from L#635 bothers my OCD!

```suggestion
          dataType.namespace !== 'auth'
```

Reply:
Removed in 0b6aec3d3a9e726d6d5422d763dd73313589f20f.